metrics/validate_drp.yaml

```diff
@@ -0,0 +1,223 @@
+# validate_drp metric definitions
+# Definitions from LPM-17 (the Science Requirements Document).
+# http://ls.st/lpm-17
+
+PA1:
+  description: >
+    The maximum rms of the unresolved source magnitude distribution around the
```

Comment:
I think we should do yaml in the "docs" way of only making newlines on new sentences, instead of enforcing a hard column limit. These are closer to docs than code, I think.